src/main/java/com/fidesmo/javacard/counter/CounterApplet.java

```diff
@@ -0,0 +1,75 @@
+package com.fidesmo.javacard.counter;
+
+import javacard.framework.*;
+
+public class CounterApplet extends Applet
+{
+    final static short noOffset = (short) 0;
+
+    final byte[] value;
```

Comment:
@miguelcardo Since you in the end only use one byte of this array I would refactor it to be just be `byte value` instead.